webextensions/manifest/permissions.json

```diff
@@ -106,7 +319,7 @@
             }
           }
         },
-        "clipboardRead": {
+        "downloads.open": {
```

Comment:
Sorry about the error message. It's really unhelpful. The issue here is that dots aren't allowed in identifiers, as oftentimes features should rather be sub features in that case and I believe this applies here as well.

So, this should be called `open` and be nested under `downloads` like this:

```json
        "downloads": {
          "__compat": {
            "support": {
              "chrome": {
                "version_added": true
              },
              "edge": {
                "version_added": false
              },
              "firefox": {
                "version_added": "47"
              },
              "firefox_android": {
                "version_added": "48"
              },
              "opera": {
                "version_added": true
              }
            }
          },
          "open": {
            "__compat": {
              "support": {
                "chrome": {
                  "version_added": true
                },
                "edge": {
                  "version_added": false
                },
                "firefox": {
                  "version_added": "48"
                },
                "firefox_android": {
                  "version_added": "48"
                },
                "opera": {
                  "version_added": true
                }
              }
            }
          }
        },
```

Reply:
Hi Florian, that resolved the errors thrown by npm test - although I still got errors when I tried to build the HTML locally using npm run render $query $dept $aggregateMode - however, the CI build doesn't seem to have thrown any errors, so hopefully we are all good. Thanks for your help.

Reply:
> So, this should be called open and be nested under downloads like this:

@Elchi3, I'm not sure about this :(. `downloads.open` is a potential value for the `permissions` key: it's a separate value from `downloads`, not a subfeature of it called `open`.

https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/permissions#API_permissions

So I'm not sure what the right thing is here. Perhaps we should call it `downloads_open` or something, and include the value in the description field? And then we should probably do the same thing for the other permissions values, to be consistent.

Reply:
Oh, I see. Sorry about this! In this case, my comment about the nesting was wrong. 

I agree to call it downloads_open, and to add a description. So it will be like this:

```json
"downloads": {
  "__compat": {
    "support": {
      "chrome": {
        "version_added": true
      },
      "edge": {
        "version_added": false
      },
      "firefox": {
        "version_added": "47"
      },
      "firefox_android": {
        "version_added": "48"
      },
      "opera": {
        "version_added": true
      }
    }
  }
},
"downloads_open": {
  "__compat": {
    "description": "<code>downloads.open</code>",
    "support": {
      "chrome": {
        "version_added": true
      },
      "edge": {
        "version_added": false
      },
      "firefox": {
        "version_added": "48"
      },
      "firefox_android": {
        "version_added": "48"
      },
      "opera": {
        "version_added": true
      }
    }
  }
}
```

Reply:
Hi, so do I need to add a description just to "download_open", or as Will seemed to suggest, to all the items?

Reply:
To me it looks like a description for `download_open` would be sufficient for now. (just like in the json I suggested above). Will should have the final say on this, though. :-)

Reply:
> To me it looks like a description for download_open would be sufficient for now. (just like in the json I suggested above). Will should have the final say on this, though. :-)

That would work, but it feels hacky to me. If `downloads.open` isn't an identifier, why are the other permission strings? It seems more consistent to do it for all the permissions, if we're going to do it for one of them.